nilearn/plotting/displays.py

```diff
@@ -218,6 +236,89 @@ def draw_position(self, size, bg_color, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def _add_graph_nodes(self, nodes_coords, node_color, node_size, **kwargs):
```

Comment:
I think that I would call this method "_add_points" (or "_add_markers"), because those points may have some meaning or some use outside the case of plotting graphs.